### PR TITLE
Fix issue-fix-dot-date-range-test

### PR DESCRIPTION
**Pull request by alfredo-dotcms**

### CODE REVIEW

File: projects/dotcms-field-elements/src/components/dot-date-range/dot-date-range.tsx [hunk 1]
```diff
@@ -142,8 +142,8 @@ export class DotDateRangeComponent {
             mode: 'range',
             altFormat: this.displayFormat,
             altInput: true,
-            maxDate: this.parseDate(this.max),
-            minDate: this.parseDate(this.min),
+            maxDate: this.max ? this.parseDate(this.max) : null,
+            minDate: this.min ? this.parseDate(this.min) : null,
```
Comment (freddyucv) [inline projects/dotcms-field-elements/src/components/dot-date-range/dot-date-range.tsx]: test for this change?

File: projects/dotcms-field-elements/src/utils/props/validators/props.ts [hunk 2]
```diff
@@ -28,6 +28,16 @@ export function dateValidator<T>(propInfo: PropValidationInfo<T>): void {
     }
 }
 
+export function dateRangeValidator<T>(propInfo: PropValidationInfo<T>): void {
```
Comment (freddyucv) [inline projects/dotcms-field-elements/src/utils/props/validators/props.ts]: doc?

Really this function not validate a Range date, I mean when I read 'dateRangeValidator' I think that tis one validate a Max and Min date but really this validate date format for two date, I don't now if the name is the right one here

File: projects/dotcms-field-elements/src/components/dot-date-range/dot-date-range.tsx (hunk 1 above)
Comment (freddyucv) [inline projects/dotcms-field-elements/src/components/dot-date-range/dot-date-range.tsx]: test?

Comment (alfredo-dotcms) [reply projects/dotcms-field-elements/src/components/dot-date-range/dot-date-range.tsx]: no idea how to test this, I mean this is added to remove the noise that's printed on the terminal when running unit test.

With or without this, there's not a difference in the DOM, so I have no idea how I could so any assert on this

Comment (fmontes) [reply projects/dotcms-field-elements/src/components/dot-date-range/dot-date-range.tsx]: Can you add a little range min and max, open the table, get all the <td> and check which one have a specific class?

File: projects/dotcms-field-elements/src/components/dot-date-range/dot-date-range.tsx [hunk 3]
```diff
@@ -119,8 +119,8 @@ export class DotDateRangeComponent {
 
     @Watch('value')
```
Comment (fmontes) [inline projects/dotcms-field-elements/src/components/dot-date-range/dot-date-range.tsx]: Can we add to the description an example, like:

```Value formatted with start and end date splitted with a comma, ex: 02-20-2012,01-10-2012```

The same should go for any other prop that have like a very specific format of date and stuff, cc: @hmoreras

**Pull request merged**